cropharvest/eo/eo.py

```diff
@@ -380,9 +334,10 @@ def _labels_to_polygons_and_years(
         print(f"Exporting {len(labels)} labels")
 
         for _, row in tqdm(labels.iterrows()):
-            latlons, bounding_box = cls._bounding_box_from_centre(
+            ee_bbox = EEBoundingBox.from_centre(
                 mid_lat=row["lat"], mid_lon=row["lon"], surrounding_metres=surrounding_metres
             )
+            latlons = (ee_bbox.min_lon, ee_bbox.min_lat, ee_bbox.max_lon, ee_bbox.max_lat)
```

Comment:
Maybe I should just pass the ee_bbox instead of latlons to ensure the ordering of the coordinates is never messed up (more robust). Thoughts?

Reply:
We manipulate latlons a fair bit. Could it be a named tuple?

but yea an easy alternative is to just add a `to_latlons` function to the bounding box. Might be best